README.md

```diff
@@ -807,15 +807,64 @@ When you have time, we invite you to take a look at:
  + [The dynamic router EIP](https://camel.apache.org/components/latest/eips/dynamicRouter-eip.html)
  + [The kamelet EIP](https://camel.apache.org/components/latest/eips/kamelet-eip.html)
 
-## Part 7 - Camel Quarkus and Kafka
+## Part 7 - Integration Example with Camel Quarkus
 Estimate time : 25 minutes
-@TODO: 25 MINUTES LEFT at this level
+
+### About this section
+In this part, we will discover how easy and fast it is to :
+* Stream data from and to Kafka
+* Write to Slack
+* Read / write to an SQL database with JPA
+* Use external APIs
+* Serialize Data
+* Create REST API
+
+
+We will also experience the joy of using [dev services](https://quarkus.io/guides/dev-services) in the development mode. No need to install any kafka server or database.
+
+---
+**🚀NOTE**
+
+This section requires having Docker installed in order to provition a Kafka and Postgresql servers. 
```

Comment:
Provision a Kafka ?

Reply:
Also, shouldn't we pre-pull some docker images in the prerequisites section ?

Reply:
yes and yes